.gitignore

```diff
@@ -10,6 +10,7 @@
 !/bin
 /bin/*
 !/bin/brew
+!/locales
```

Comment:
Personally, I think it's better to put `locales` inside `Library` as it should be part of Homebrew Library.